libs/timeserieslib/TimeSeriesValidator.h

```diff
@@ -40,6 +42,7 @@ namespace mkc_timeseries
         private:
             std::shared_ptr<OHLCTimeSeries<Decimal>> mHourlyTimeSeries;
             std::shared_ptr<OHLCTimeSeries<Decimal>> mDailyTimeSeries;
+            int numberTimeFrames;
```

Comment:
I don't see where you use 'numberTimeFrames'? It appears you still have "7" hardcoded as the number of time frames in the "mValidateSeventTimePeriods" method (which should be renamed if we don't have a fixed number of hourly bars).

Reply:
Sorry about that, I must have gotten ahead of myself.